initialize_database.py

```diff
@@ -23,4 +24,7 @@ def initialize_database(database_name="new_database.sqlite"):
 
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('filename',help= "The name of the new database")
+    parser.parse_args()
     initialize_database(sys.argv[1])
```

Comment:
If you're using `argparse`, you shouldn't read the arguments from `sys.argv` anymore but rather,

```python
initialize_database(parser.filename)
```

Ditto for `migrate_database.py`